src/fairseq2/models/conformer/convolution.py

```diff
@@ -30,6 +33,7 @@ def __init__(
         model_dim: int,
         depthwise_kernel_size: int,
         *,
+        normalization: str = "batch_norm",
```

Comment:
Nit: I would name this `norm_type` since we typically use "norm" as prefix/suffix for normalization related parameters.